packages/volto/news/6159.breaking

```diff
@@ -0,0 +1,4 @@
+Fixed image widget position and look and feel in sidebar @ichim-david
+
+Breaking:
+- This change updated the markup of the widget in the generated form both in sidebar and in a normal form.
```

Comment:
Is there a process that needs to be added to the upgrade guide?

What should developers search for and change, if needed?

In which file should they search for details?

Reply:
> Is there a process that needs to be added to the upgrade guide?
> 
> What should developers search for and change, if needed?
> 
> In which file should they search for details?

@stevepiercy I modified the text with some information on what changed.
It has todo with the image widget added ast part of Volto 18 alpha 35 or 36
https://github.com/plone/volto/pull/5607
We found some bugs that needed fixing. I don't know if we need such breaking
for breaking changes, from my point of view being in alpha means that we can
make a fix that changes things added in a preview alpha if we found problems with
the work that we approved and merged then.

If you think otherwise and still need further info added get in touch with me and lets
finish the info as needed.

Reply:
@stevepiercy @ichim-david Since it's a breaking of a breaking and it's about markup, it's fine. We only did that in case an API changing, etc.

Reply:
OK, that sounds good, just wanted to make sure that breaking is OK in this context.